app/views/pages/personal_dashboard.html.erb

```diff
@@ -1,25 +1,21 @@
 <div class="container">
   <h1>Personal dashboard</h1>
 
-  <!-- list users details -->
   <div class="personal-details">
-
-    <h2>Account</h2>
-    <img src="" alt="avatar" class="rounded-circle">
-    <%= current_user.first_name %>
-    <%= current_user.last_name %>
-    <%= current_user.address %>
-    <%= current_user.email %>
-    <button type="button" class="btn"></button>
-    <%= link_to 'edit', edit_user_registration_path %>
-
-    <button type="button" class="btn"></button>
-    <%= link_to 'delete', cancel_user_registration_path %>
+    <h2>Account</h2><hr>
+    <img src="" alt="avatar" class="rounded-circle"><hr>
+    <%= current_user.first_name %><br>
+    <%= current_user.last_name %><br>
+    <%= current_user.address %><br>
+    <%= current_user.email %><br><br>
+    <button type="button" class="btn btn-primary">Edit</button>
```

Comment:
why do we have a button here and a link to in the next line with no destination?

Reply:
get rid of <button> and link_to,
replace with:
<%= link_to 'Edit', edit_user_registration_path, class: "btn btn-primary" %>